saleor/graphql/product/mutations/channels.py

```diff
@@ -318,8 +318,6 @@ def remove_variants(
         ).exists():
             product_channel_listing.delete()
 
-        cls.perform_checkout_lines_delete(variants, [channel.id])
```

Comment:
Should you remove the body of function?

Reply:
@fowczarek I can't right now. It is still used when removing ProductChannelListing. So need to wait till applying similar changes for product-channel-listing